packages/desktop-client/src/components/common/Select.tsx

```diff
@@ -61,6 +63,7 @@ export function Select<const Value = string>({
     <>
       <Button
         ref={triggerRef}
+        id={id}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏why do we need an `id` on the buttons? And what does it do?

Reply:
It's used for the `htmlFor` attributes that were pre-existing in `RecurringSchedulePicker`. The `id` attribute was actually being passed as a prop but the labels weren't being correctly associated with the fields. I suppose you can consider this an additional bugfix caught by static typing since they were supposed to be associated?

Reply:
Ah.. ok, that makes sense for input fields. But not so much for buttons I think.

Reply:
Which element should we put it on here? The `htmlFor` is supposed to be for an input element IIRC, but the "input" in this case is the `Select`.

Reply:
🤦 ignore my previous comment. I'm reviewing too many things in parallel.

--

It makes sense to put it on the `Button` here since the general element is `Select` and the button is used for opening the select popover.

Reply:
no worries!